imports/blocks/give/__tests__/__snapshots__/Layout.js.snap

```diff
@@ -992,16 +992,6 @@ exports[`test renders Success 1`] = `
       Success!
     </h3>
     <p
-      className="text-left">
```

Comment:
@richarddubay I'd like to keep a snapshot of the schedule layout. Can we mock moment instead to print out the date + formatting like we do in other tests (i.e. [this](https://github.com/NewSpring/Holtzman/blob/give-routing/imports/blocks/add-to-cart/Schedule/__tests__/index.js#L15-L25) for [this](https://github.com/NewSpring/Holtzman/blob/give-routing/imports/blocks/add-to-cart/Schedule/index.js#L30-L35).